cerberus/custom_checks/custom_checks.py

```diff
@@ -90,6 +91,26 @@ def check_PV():
         return False
 
 
+def check_kyverno():
+    logging.info("Check if the Kyverno ConfigMap is okay.")
+    result = False
+
+    create_config = runcommand.invoke(
+        'oc create -n kyverno configmap simple-test --from-literal=data="asdf" && echo "successed"')
```

Comment:
Lets create the cm in the cerberus namespace instead of the kyverno one

Reply:
how is kyverno test related to a configmap create/update? is there a healthiness endpoint we can add? can we have some doc/comment to explain?

Reply:
Kyverno is validating configmap create / update / delete operations. We're doing "black box" testing of Kyverno instead of just checking its health. This also indirectly checks the API in general for other non-kyverno issues.